docs/schema-registry.md

```diff
@@ -0,0 +1,33 @@
+# Configuration Schema Registry
+
+Although we support the use of schemas without external dependencies such as Schema Registry, it is common to use it, since it allows to
+have a control of the schemas with versions.
+
+To choose the use of Schema Registry it is necessary to do it in the following way introducing the basic properties that appear in the
+image:
+![](images/SchemaRegistrySelection.png)
+
+The properties are supported by the schema registry available to us (At the moment are Confluent Schema Registry, and Apicurio Schema Registry)
+
+A list of the properties explained here:
+* `schema.registry.auth.enabled` If your schema registry have auth enabled or not
+* `schema.registry.auth.method` The method to connect with your schema registry
+* `schema.registry.username` Username
+* `schema.registry.password` Password
+* `schema.registry.bearer`	Token for SASL/OAUTHBEARER
+
+## Some Recommendations
+
+To avoid having unclear schemas, when use Apicurio they recommend using this functionality instead of auto-generating.
+https://www.apicur.io/registry/docs/apicurio-registry/2.1.x/assets-attachments/registry-rest-api.htm#tag/Artifacts/operation/createArtifact
+
+In Confluent Schema Registry instead they take the SubjectName as ID, so remember this if you see a different name in your Schema List
```

Comment:
`instead of`

Reply:
I think I will change that sentence for this one "However, in Confluent Schema Registry, they take the SubjectName as ID, so remember it if you see a different name in your Schema List" because the sense of the first one was not the desired one. Let me know if you like this one too

Reply:
I don't understand what you want to say.
It seems that you are giving tips to make the Apicurio schema.
And I don't have the context to understand it, another reason is that I can't find references to `SubjectName` or `ID` in this file.
So, if you think that someone with the right knowledge can understand it, go ahead.

Reply:
I don't think that putting there an explanation about Schema Registry is the goal, I think we should put just an explanation about how to configure it in our plugin, if people who intend to use Schema Registry in our plugin don't have knowledge about it, they should check first the documentation of the Schema Registry they want to use before reading this section, if anything we could add links to the documentation of Confluent and Apicurio in case they don't know it and don't know where to look

How would you see that?